guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
+
+#### Guild#voice
+
+The `Guild#voice` getter has been removed.
+
+```diff
+- guild.voice
++ guild.me.voice
+```
+
+### GuildChannel
+
+#### GuildChannel#createOverwrite
+
+This method has been removed, with functionality replaced by the new `PermissionOverwriteManager`.
+
+```diff
+- channel.createOverwrite(user, { VIEW_CHANNEL: false });
++ channel.permissionOverwrites.create(user, { VIEW_CHANNEL: false });
+```
+
+#### GuildChannel#overwritePermissions
+
+This method has been removed, with functionality replaced by the new `PermissionOverwriteManager`.
+
+```diff
+- channel.overwritePermissions([{ id: user.id , allow: ['VIEW_CHANNEL'], deny: ['SEND_MESSAGES'] }]);
++ channel.permissionOverwrites.set([{ id: user.id , allow: ['VIEW_CHANNEL'], deny: ['SEND_MESSAGES'] }]);
+```
+
+#### GuildChannel#permissionOverwrites
+
+This method no longer returns a Collection of PermissionOverwrites, instead providing access to the `PermissionOverwriteManager`.
+
+#### GuildChannel#updateOverwrite
+
+This method has been removed, with functionality replaced by the new `PermissionOverwriteManager`.
+
+```diff
+- channel.updateOverwrite(user, { VIEW_CHANNEL: false });
++ channel.permissionOverwrites.edit(user, { VIEW_CHANNEL: false });
+```
+
+### GuildMember 
+
+#### GuildMember#ban
+
+`GuildMember#ban()` will throw a TypeError when a string is provided instead of an options object.
+
+```diff
+- member.ban('reason')
++ member.ban({ reason: 'reason' })
+```
+
+### GuildMember#hasPermission
+
+This `GuildMember#hasPermission` shortcut/helper method has been removed.
+
+```diff
+- member.hasPermission(Permissions.FLAGS.SEND_MESSAGES);
++ member.permissions.has(Permissions.FLAGS.SEND_MESSAGES);
+```
+
+### GuildMemberManager
+
+#### GuildMemberManager#ban
+
+The `GuildMemberManager#ban` method will throw a TypeError when a string is provided instead of an options object.
+
+```diff
+- guild.members.ban('123456789012345678', 'reason')
++ guild.members.ban('123456789012345678', { reason: 'reason' })
+```
+
+### Message / MessageManager
+
+#### Message#delete
+
+The `Message.delete()` method no longer accepts any options, requiring a timed-delete to be performed manually.
+
+```diff
+- message.delete({ timeout: 10000 });
++ client.setTimeout(() => message.delete(), 10000);
+```
+
+`reason` is no longer a parameter as it is not used by the API.
+
+#### MessageManager#delete
+
+The `MessageManager.delete()` method no longer accepts any additional options, requiring a timed-delete to be performed manually.
+
+```diff
+- channel.messages.delete('123456789012345678', { timeout: 10000 });
++ client.setTimeout(() => channel.messages.delete('123456789012345678'), 10000);
+```
+
+`reason` is no longer a parameter as it is not used by the API.
+
+### MessageEmbed
+
+#### MessageEmbed#attachFiles
+
+The `MessageEmbed#attachFiles` method has been removed. Instead, files should be attached to the Message directly via MessageOptions.
```

Comment:
```suggestion
The `MessageEmbed#attachFiles` method has been removed. Instead, files should be attached to the Message directly via `MessageOptions`.
```

Reply:
There are many words referenced as types (Guild, Channel, Message...) that aren't marked as inline codes, not sure if that's intentional.

Reply:
There are times when I've treated it more like a proper noun as the name of a thing, rather than a type. Maybe I should just lower case those to regular nouns. This is definitely a type though.